mediawiki/LocalSettings.d/LocalSettings.override.php

```diff
@@ -143,5 +143,10 @@
 $wgMathFullRestbaseURL = 'https://wikimedia.org/api/rest_';
 $wgMathMathMLUrl = 'https://mathoid-beta.wmflabs.org';
 
+#popups for math
```

Comment:
```suggestion
# Settings for math-extension: popups for math
```

Reply:
Suggestion to add clarifying comment, so it is more obvious where settings are found.